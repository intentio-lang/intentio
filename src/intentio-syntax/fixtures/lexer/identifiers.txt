_ignored
intentio
inte_ntio'
qëllim
意向
მიზანი
इरादा
_इरादा_
इरादा'
