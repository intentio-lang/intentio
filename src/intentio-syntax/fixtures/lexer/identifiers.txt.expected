Ident "_ignored"
Ident "intentio"
Ident "inte_ntio'"
Ident "q\235llim"
Ident "\24847\21521"
Ident "\4315\4312\4310\4304\4316\4312"
Ident "\2311\2352\2366\2342\2366"
Ident "_\2311\2352\2366\2342\2366_"
Ident "\2311\2352\2366\2342\2366'"
